onchain/src/art_peace.cairo

```diff
@@ -91,6 +92,13 @@ pub mod ArtPeace {
         color: u8,
     }
 
+    #[derive(Drop, starknet::Event)]
+    pub struct QuestClaimed {
+        pub user: ContractAddress,
```

Comment:
Could you change this to 2 seperate events?
`DailyQuestClaimed` and `MainQuestClaimed`, then add the `quest_id` and `day_idx` ( for daily quests ) fields to each since that is needed now to distinguish between quests.

Reply:
yes i was about to ask you, found weird to emit the same event for daily quests and mainquests, perfect!

Reply:
done